styleguide/core/Input.hbs

```diff
@@ -22,8 +22,8 @@
         <div class="{{blockName}}"{{extraAttributes}}>
 
             {{#defineBlockBody}}
-                {{element "input"}}
                 {{element "label"}}
+                {{element "input"}}
```

Comment:
You certainly could, assuming your parent selector wraps a single input of the type that you're looking to style the label specific to. But that's not what I'm trying to solve for. I'd just like to know whether you guys think it makes more sense to have the template be the source of truth when it comes to the type of input we're rendering. (I think 😄 ) The other issues would fall away as part of that change.